aea/cli/remove.py

```diff
@@ -322,11 +315,9 @@ def __init__(
         :param ctx: click context.
         :param item_type: str, package type
         :param item_id: PublicId of the item to remove.
+        :param with_dependencies: whether or not to remove dependencies.
         :param force: bool. if True remove even required by another package.
-        :param ignore_non_vendor: bool. if True, ignore non-vendor packages when computing
-          inverse dependencies. The effect of this flag is ignored if force = True
-
-        :return: None
+        :param ignore_non_vendor: bool. if True, ignore non-vendor packages when computing inverse dependencies. The effect of this flag is ignored if force = True
```

Comment:
a bit long row? Maybe split if it is allowed

Reply:
Not trivial to split rows now for some reason. For now keeping it as one line.